docs/cobbler.rst

```diff
@@ -790,12 +790,81 @@ Add enabled yum repositories from ``dnf repolist --enabled`` list. The repositor
 Cobbler image
 =============
 
+The primary and recommended use of cobbler is to deploy systems by building them, as though from the OS manufacturer's
+distribution, e.g Redhat kickstart. This method is generally easier to work with and provides an infrastructure which is
+not only more sustainable but also much more flexible across varieties of hardware.
+
+But Cobbler can help with image-based booting, physically and virtually. Some manual use of other commands beyond what
+is typically required of cobbler may be needed to prepare images for use with this feature, and the usage of these
+commands varies substantially by the type of image.
+
+There is a lot more to go here. For the moment we just have 1 example of using the "memdisk" image type
+
 Example:
 
 .. code-block:: shell
 
     $ cobbler image
 
+memdisk - Oracle / Sun Maintenance CD
+-------------------------------------
+
+The 'memdisk' image type can be used to PXE boot Oracle / Sun maintenance CDs.
+`Their manual <http://docs.oracle.com/cd/E19121-01/sf.x2250/820-4593-12/AppB.html#50540564_72480>`_ gives details on how
+to copy the boot CD to a PXE server. The procedure is even easier with cobbler since the system takes care of most of
+that stuff for you.
```

Comment:
```suggestion
to copy the image from a CD to a PXE server. The procedure is even easier with Cobbler since the system takes care of most of
it for you.
```